docs/about-ensembl/the-ensembl-project/cite.md

```diff
@@ -0,0 +1,24 @@
+---
+slug: cite
+title: Citing Ensembl
+description:
+status: draft
+---
+
+# Citing Ensembl
+
+If you've used Ensembl in your work, please cite the appropriate overview article:
+
+## For vertebrate genomes
+
+*Ensembl 2020*\
```

Comment:
Single asterisk in markdown is the same as an underscore (translates to the `em` tag).

If your intention was to make this text bold, you need two asterisks on both sides of the word.

Reply:
Oops! I use Markdown all the time in Joplin, but today my brain is not working...